helix-core/src/main/java/org/apache/helix/task/JobRebalancer.java

```diff
@@ -154,6 +154,7 @@ public ResourceAssignment computeBestPossiblePartitionState(ClusterDataCache clu
     // Update rebalancer context, previous ideal state.
     TaskUtil.setJobContext(_manager, jobName, jobCtx);
     TaskUtil.setWorkflowContext(_manager, workflowResource, workflowCtx);
+    TaskUtil.createUserContentNode(_manager, jobName, new ZNRecord(TaskUtil.USER_CONTENT_NODE));
```

Comment:
Put the create call here will make it called everytime rebalance() is executed, that should be avoided.  Create should be only happen once, we can create the user content node when workflowRebalancer creates the job's idealState creation.